### DISCUSSION TITLE
Welcome to GNN4NLP-Papers Discussions!

### DISCUSSION THREAD

**Discussion by IndexFziQ** [IndexFziQ/GNN4NLP-Papers#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WXB-Wilber** [comment]
This is an useful share, why not updata it? please.

**Reply by IndexFziQ** [reply]
Thanks for your support! I will update some GNN4NLP papers of LLM era. I am busy lately😣
